src/hooks/web3/useBridgeSommToEthTx.tsx

```diff
@@ -161,49 +101,107 @@ export const useBridgeSommToEthTx = () => {
           },
         },
       })
-      if (res.code !== 0) {
-        analytics.track("bridge.contract-failed", {
+
+      if (res.transactionHash && res.code === 0) {
+        analytics.track("bridge.contract-succeeded", {
           value: props.amount,
           path: "sommToEth",
           sender: data?.bech32Address,
           receiver: props.address,
           txHash: res.transactionHash,
         })
-        setIsLoading(false)
-        return update({
+
+        update({
           heading: "Bridge Initiated",
           body: (
-            <TxHashToastBody
-              title="Transaction Failed"
-              hash={res.transactionHash}
-            />
+            <Stack>
+              <Stack spacing={0} fontSize="xs">
+                <HStack>
+                  <Text as="span" fontWeight="bold">
+                    Amount:
+                  </Text>
+                  <Text as="span">{props.amount} SOMM</Text>
+                </HStack>
+                <HStack>
+                  <Text as="span" fontWeight="bold">
+                    Destination:
+                  </Text>
+                  <Text as="span">Sommelier to Ethereum Mainnet</Text>
+                </HStack>
+                <HStack>
+                  <Text as="span" fontWeight="bold" width="15ch">
+                    Est. time:
+                  </Text>
+                  <Text as="span">
+                    1-5 min. Transaction may take additional time to
+                    process after network validation
+                  </Text>
+                </HStack>
+              </Stack>
+              <Link
+                fontSize="sm"
+                href={`https://www.mintscan.io/sommelier/txs/${res.transactionHash}`}
+                target="_blank"
+                textDecor="underline"
+              >
+                <HStack>
+                  <Text>View on Mintscan</Text>
+                  <ExternalLinkIcon boxSize={3} />
+                </HStack>
+              </Link>
+            </Stack>
           ),
-          status: "error",
+          status: "primary",
           duration: null,
           closeHandler: closeAll,
         })
-      }
-      if (res.transactionHash && res.code === 0) {
-        analytics.track("bridge.contract-succeeded", {
+
+        // Trigger the import token functionality
```

Comment:
What if the user already has the token imported? Doing it every bridge tx seems like a bit much, a button seems better